modules/vba_documents/app/models/vba_documents/monthly_stat.rb

```diff
@@ -3,6 +3,6 @@
 module VBADocuments
   class MonthlyStat < ApplicationRecord
     validates :month, inclusion: { in: (1..12).to_a }, presence: true, uniqueness: { scope: :year }
-    validates :year, format: { with: /\A(?:19|20)\d{2}\z/ }, presence: true
+    validates :year, format: { with: /\A2\d{3}\z/ }, presence: true
```

Comment:
Since the Benefits Intake API launched in 2018, I realized that a regex for 2000-2999 would be more than sufficient.